README.md

```diff
@@ -27,6 +27,6 @@ To help triage or fix bugs, see the current [Ignition issues](https://github.com
 To validate a config for Ignition there are binaries for a cli tool called ignition-validate available [on the releases page][releases], and an online validator available [on the CoreOS website][online-validator].
 
 [getting started]: doc/getting-started.md
-[issues]: https://github.com/coreos/bugs/issues/new?labels=component/ignition
+[issues]: https://github.com/coreos/ignition/issues/new
```

Comment:
If https://github.com/coreos/ignition/pull/595 merges, this should be https://github.com/coreos/ignition/issues/new/choose.